.github/workflows/ci.yaml

```diff
@@ -22,6 +22,16 @@ jobs:
           go-version: '^1.19.4'
       - run: make test
 
+  integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.19.4'
+      - run: make rollout-operator build-image
```

Comment:
Why do you also need to make the `build-image` here?

Reply:
I need the image for the integration tests. Maybe there's something obvious I'm missing?

Reply:
Update after an offline discussion. `build-image` builds the `rollout-operator` image and not a "build image" image (like in Mimir). I got confused by it. Maybe we don't need `make rollout-operator` tho.

Reply:
Removed `rollout-operator` target in https://github.com/grafana/rollout-operator/pull/30/commits/449a7d9871d118ab5189bec997881668ff090f66